### ISSUE TITLE
Private repos

### ISSUE THREAD

**Issue by MaffooBristol** [seriema/npmalerts#8]
Any plans for giving access to github private repos? Or even allowing the upload of a package.json file?

Cheers

**Comment by seriema** [comment]
I'd like to do a real Github authentication thingie, so you can just select your github repos. I didn't do that at first because I had the idea to keep npmalerts separate from a specific repo, and allow things like you suggested (upload of package.json). But now I think adding a Github connection would be more useful. Do you agree?

**Comment by MaffooBristol** [comment]
Definitely, especially if you could easily select all your repos, or all your source repos (ie, the ones you've created not forked), etc!

Thanks for making this tool; I got my first notification from it today so I see it's working! :smile: How often is cron run btw? Daily? What time?

**Comment by seriema** [comment]
Thanks for the input. I'll try to look into it soon!

Yeah it runs once every night, around 02 CET I think.
